js/pro/huobi.js

```diff
@@ -89,6 +89,7 @@ module.exports = class huobi extends huobiRest {
                 'ws': {
                     'gunzip': true,
                 },
+                'watchTickerTopic': 'market.{marketId}.detail', // 'market.{marketId}.bbo' or 'market.{marketId}.ticker'
```

Comment:
@pcriadoperez usually we use a different notation here, something like:

```
watchTicker: {
     'name': 'market.{marketId}.detail'
}
```

Can you change it for the sake of consistency please?

Reply:
@pcriadoperez btw I think that `market.{marketId}.ticker` is only valid for spot markets, so should we add a check inside `watchTickers` validating it?

```
 n huobi watchTicker "BTC/USD:BTC"
Debugger attached.
2022-12-25T10:33:58.470Z
Node.js: v17.9.0
CCXT v2.4.27
huobi.watchTicker (BTC/USD:BTC)
BadRequest {"id":"1","ts":1671964441449,"status":"error","err-code":"bad-request","err-msg":"invalid topic market.BTC-USD.ticker"}
---------------------------------------------------
[BadRequest] {"id":"1","ts":1671964441449,"status":"error","err-code":"bad-request","err-msg":"invalid topic market.BTC-USD.ticker"}

    at throwExactlyMatchedException  js/base/Exchange.js:2353    throw new exact[string] (message);                                              
    at handleErrorMessage            js/pro/huobi.js:1711        this.throwExactlyMatchedException (this.exceptions['ws']['exact'], errorCode, t…
    at handleMessage                 js/pro/huobi.js:1745        if (this.handleErrorMessage (client, message)) {                                
    at onMessage                     js/pro/base/Client.js:283   this.onMessageCallback (this, message)                                          
    at onMessage                     ws/lib/event-target.js:199  listener.call(this, event);                                                     
    at emit                          node:events:527                                                                                             
    at receiverOnMessage             ws/lib/websocket.js:1178    this[kWebSocket].emit('message', data, isBinary);                               
    at emit                          node:events:527                                                                                             
    at dataMessage                   ws/lib/receiver.js:513      this.emit('message', data, true);                                               
    at getData                       ws/lib/receiver.js:446      return this.dataMessage();                                                      

{"id":"1","ts":1671964441449,"status":"error","err-code":"bad-request","err-msg":"invalid topic market.BTC-USD.ticker"}
```